.semgrep.yml

```diff
@@ -0,0 +1,32 @@
+rules:
+  - id: use-pure-path
+    patterns:
+      - pattern: pathlib.Path
```

Comment:
So how would this match against something like you did: `from pathlib import Path`?

Reply:
yeah, semgrep is pretty nifty. It understands the control flow/imports (as long as they're not trying to be actively misleading, of course), e.g. https://semgrep.dev/playground/s/xAzA

```python
import pathlib as deceptive
from pathlib import Path
from pathlib import Path as AlsoDeceptive

Path("x")
deceptive.Path("y")
AlsoDeceptive("z")
```

All three of them are flagged.